features/dd-sdk-android-rum/src/test/kotlin/com/datadog/android/rum/internal/vitals/JankStatsActivityLifecycleListenerTest.kt

```diff
@@ -356,10 +359,41 @@ internal class JankStatsActivityLifecycleListenerTest {
         verify(mockObserver, never()).onNewSample(any())
     }
 
+    @Test
+    fun `𝕄 adjust sample value to refresh rate 𝕎 doFrame() {refresh rate over 60hz}`(
```

Comment:
ideally, it would be nice to see the tests for different Android API versions (since there are different code paths in the class under test depending on API version), but this may explode the number of tests, so it is not a blocker for me.

Reply:
We just need to test pre-R, R and post-R. 
I can include those.